industrial_ci/src/env.sh

```diff
@@ -109,6 +92,31 @@ if [ -z "$OS_CODE_NAME" ]; then
         ;;
     esac
 fi
+if [ "$ROS_VERSION_EOL" = true ]; then
+  if [ ! "$HASHKEY_SKS" ]; then export HASHKEY_SKS="AD19BAB3CBF125EA"; fi
+  ROS_REPO_SERVER="http://snapshots.ros.org"
+else
+  if [ ! "$HASHKEY_SKS" ]; then export HASHKEY_SKS="C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654"; fi
+  ROS_REPO_SERVER="http://packages.ros.org"
+fi
+
+# If not specified, use ROS Shadow repository http://wiki.ros.org/ShadowRepository
+if [ ! "$ROS_REPOSITORY_PATH" ]; then
```

Comment:
Should there be any special handling of the EOL distro case here? Is it reasonable to expect `ros` and `ros-testing` to be present in the snapshots server as well? I guess it may be, since there are still EOL'd distro packages in the current `ros-shadow-fixed` repo.

Reply:
I don't know yet. I assume that all of them will get renamed (the repos get signed with the new key anyway).

Reply:
Ah, the layout changes completely, e.g. http://snapshots.ros.org/cturtle/final/ubuntu/
I will adapt the logic.

Reply:
> I will adapt the logic.

Done.
In addition non-final snapshot can be used as well